src/test/java/org/araymond/joal/core/client/emulated/generator/key/TimedOrAfterStartedAnnounceRefreshKeyGeneratorTest.java

```diff
@@ -19,22 +19,13 @@
  */
 public class TimedOrAfterStartedAnnounceRefreshKeyGeneratorTest {
 
-    @Test
-    public void shouldNotBuildWithoutRefreshEvery() {
-        final KeyAlgorithm algo = Mockito.mock(KeyAlgorithm.class);
-        Mockito.when(algo.generate()).thenReturn("do-not-care");
-        assertThatThrownBy(() -> new TimedOrAfterStartedAnnounceRefreshKeyGenerator(null, algo, Casing.NONE))
-                .isInstanceOf(TorrentClientConfigIntegrityException.class)
-                .hasMessage("refreshEvery must be greater than 0.");
-    }
-
```

Comment:
This test and many similar were removed as we converted from boxed types to primitives, hence null is no longer an option.